cmd/db/tss_migrations.go

```diff
@@ -0,0 +1,95 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	tssmigrations "github.com/stellar/stellar-disbursement-platform-backend/db/migrations/tss-migrations"
+)
+
+type TSSDatabaseMigrationManager struct {
+	RootDatabaseDSN      string
+	RootDBConnectionPool db.DBConnectionPool
+}
+
+func (m *TSSDatabaseMigrationManager) SchemaName() string {
+	return "tss"
+}
+
+func NewTSSDatabaseMigrationManager(rootDatabaseDSN string) (*TSSDatabaseMigrationManager, error) {
+	rootDBConnectionPool, err := db.OpenDBConnectionPool(rootDatabaseDSN)
+	if err != nil {
+		return nil, fmt.Errorf("opening root database connection pool: %w", err)
+	}
+
+	return &TSSDatabaseMigrationManager{
+		RootDatabaseDSN:      rootDatabaseDSN,
+		RootDBConnectionPool: rootDBConnectionPool,
+	}, nil
+}
+
+func (m *TSSDatabaseMigrationManager) createTSSSchemaIfNeeded(ctx context.Context) error {
+	query := fmt.Sprintf("CREATE SCHEMA IF NOT EXISTS %s", m.SchemaName())
```

Comment:
Do we want to create a schema for the `tss`?

Reply:
It could also be in the public schema, but we decided to have it in its own schema because we can have the TSS being a standalone project if we do that.